examples/data-objects/codemirror/src/index.ts

```diff
@@ -57,16 +37,27 @@ class CodeMirrorFactory extends RuntimeFactoryHelper {
 			[smde.type, Promise.resolve(smde)],
 		]);
 
-		const runtime: ContainerRuntime = await ContainerRuntime.load(
+		const runtime: ContainerRuntime = await ContainerRuntime.loadRuntime({
 			context,
-			registry,
-			buildRuntimeRequestHandler(
-				mountableViewRequestHandler(MountableView, [viewRequestHandler]),
-			),
-			undefined, // runtimeOptions
-			undefined, // containerScope
+			registryEntries: registry,
 			existing,
-		);
+			initializeEntryPoint: async (containerRuntime: IContainerRuntime) => {
+				// ISSUE: IContainerRuntime doesn't have methods that expose data stores as IDataStore or
+				// IFluidDataStoreChannel, which expose entryPoint. getRootDataStore returns an IFluidRouter.
+				const dataStore: IFluidDataStoreChannel = (await containerRuntime.getRootDataStore(
+					defaultComponentId,
+				)) as IFluidDataStoreChannel;
+
+				// TODO: better type discovery
+				const codeMirror: CodeMirrorComponent =
+					(await dataStore.entryPoint?.get()) as CodeMirrorComponent;
+				if (codeMirror === undefined) {
+					throw new Error("DataStore did not set its EntryPoint");
+				}
+
+				return new CodeMirrorView(codeMirror.text, codeMirror.presenceManager);
+			},
+		});
```

Comment:
Container runtime factory now sets an entryPoint for the Container, which exposes the entryPoint of its default root data store.